plugin/node.js

```diff
@@ -129,7 +134,12 @@
   }
 
   function postLoadDef(data) {
-    var cx = infer.cx(), mods = cx.definitions[data["!name"]]["!node"];
+    var cx = infer.cx();
+    var def = cx.definitions[data["!name"]];
+    if (!def || !def["!node"]) {
+      return;
+    }
+    var mods = def["!node"];
```

Comment:
just for debugging. It was hard to step through these lines in the node debugger. I can revert them if you like.